cookienotice/cookienotice.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * Name: Cookie Notice
+ * Description: Configure, show and handle a simple cookie notice
+ * Version: 1.0
+ * Author: Peter Liebetrau <https://socivitas/profile/peerteer>
+ * 
+ */
+use Friendica\Core\Addon;
+use Friendica\Core\Config;
+use Friendica\Core\L10n;
+
+function cookienotice_install()
+{
+	$file = 'addon/cookienotice/cookienotice.php';
+	Addon::registerHook('page_content_top', $file, 'cookienotice_page_content_top');
+	Addon::registerHook('page_end', $file, 'cookienotice_page_end');
+	Addon::registerHook('addon_settings', $file, 'cookienotice_addon_settings');
+	Addon::registerHook('addon_settings_post', $file, 'cookienotice_addon_settings_post');
+}
+
+function cookienotice_uninstall()
+{
+	$file = 'addon/cookienotice/cookienotice.php';
+	Addon::unregisterHook('page_content_top', $file, 'cookienotice_page_content_top');
+	Addon::unregisterHook('page_end', $file, 'cookienotice_page_end');
+	Addon::unregisterHook('addon_settings', $file, 'cookienotice_addon_settings');
+	Addon::unregisterHook('addon_settings_post', $file, 'cookienotice_addon_settings_post');
+}
+
+function cookienotice_addon_settings(&$a, &$s)
+{
+	if (!is_site_admin())
```

Comment:
Code standards: Please add braces around all conditional structures.

Reply:
I' with you and I also dont like "missing braces". Since I "stole" this code from the pageheader plugin, I am not that guilty. :D
Will fix these things.

Reply:
done